alf/drivers/policy_driver.py

```diff
@@ -113,12 +113,20 @@ def get_initial_time_step(self):
                                                      self.env.batch_size)
         return make_action_time_step(time_step, action)
 
-    def algorithm_step(self, time_step, state):
+    def algorithm_step(self, time_step, state, training=self._training):
         if self._observation_transformer is not None:
             time_step = time_step._replace(
                 observation=self._observation_transformer(time_step.
                                                           observation))
-        return self._algorithm_step(time_step, state)
+        return self._algorithm_step(time_step, state, training)
+
+    def _algorithm_step(self, time_step, state, training):
```

Comment:
if _algorithm_step() is only used by algorithm_step(), we should merge them into one single function.

Reply:
merged